OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -36,6 +36,15 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name:
 
+deployment:
+  # Uncomment and modify this to run a command before starting the IQ Server
+  preStart:
+    command:    # '["/bin/rm", "-f", "/path/to/lockfile"]'
```

Comment:
I forget what we did with the other one. Can we get concrete paths to the lock file, as that might be the most common case? Or do we not yet want to encourage this behavior yet?

Reply:
The path to the logfile is version-dependent and complicated. It's certainly not obvious at a glance what it is or why it's there. Not to mention it's something I don't want to try to maintain. I would rather provide an example that is clear to read even if it is not immediately usable.